tools/filter_cells_and_genes/filter_cells_and_genes.xml

```diff
@@ -63,11 +68,18 @@ Rscript $__tool_directory__/filter_cells_and_genes.R
          <option value="Seurat">Seurat</option>
       </param>
       <when value="scater">
-         <param name="gene_filter" type="select" label="How do you want to filter out genes ?">
-            <option value="low.abundances" selected="true">low abundances genes relative to library size</option>
-            <option value="min.cells">genes that aren't detected in a fraction of cells</option>
-         </param>
-         <param name="min_cells" type="integer" value="3" label="What threshold do you want to apply ?"/>
+         <conditional name="feature_removing">
+            <param name="gene_filter" type="select" label="How do you want to filter out genes ?">
+               <option value="low.abundances" selected="true">low abundances genes relative to library size</option>
+               <option value="min.cells">genes that aren't detected in a fraction of cells</option>
+            </param>
+            <when value="low.abundances">
+               <param name="min_counts" type="integer" value="1" label="Keep all genes with an average count superior to:"/>
```

Comment:
Keep all genes with an average count higher than:

By the way, the default parameter value is 1. does that mean that under the default setting, only genes with mean count equal or higher than `2` are kept ?

Reply:
OK, thanks. Thus the label `Keep all genes with an average count superior to:` was misleading too right ?